onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -167,6 +169,17 @@ TensorrtExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph,
     result.push_back(std::make_unique<ComputeCapability>(std::move(sub_graph)));
   }
 
+  // Fallback to CUDA if the nodes are not supported by TensorRT
+  for (int i = 0, end = graph.NumberOfNodes(); i < end; ++i) {
+    if (supported_nodes_set.find(i) == supported_nodes_set.end()) {
+      const auto& node = graph.GetNode(i);
+      //LOGS_DEFAULT(WARNING) << "Fallback to CPU execution provider for Op type: " << node->OpType() << " node name: " << node->Name();
+      auto update_node = const_cast<Node*>(node);
+      //update_node->SetExecutionProviderType(onnxruntime::kCpuExecutionProvider);
+      update_node->SetExecutionProviderType(onnxruntime::kCudaExecutionProvider);
```

Comment:
how can you guarantee the node is supported by cuda provider?

also it feels like we are breaking the api here. GetCapability is supposed to return whether this provider is capable of executing the node. (its purpose is not to say anything about the capabilities of other providers)

Reply:
removed this code since it's not needed